.github/workflows/test.yml

```
@@ -110,55 +110,55 @@ jobs:
-  #         retention-days: 7
-  #         status: ${{ job.status }}
-  #         token: ${{ secrets.CODECOV_TOKEN }}
-  #         codecov-files: examples-partcad-test-results/junit.xml
-  #         coverage-report: examples-partcad-test-results/coverage.xml
+  test-examples-partcad:
+    name: "Examples (PartCAD)"
+    strategy:
+      fail-fast: false
+      matrix:
+        # TODO(clairbee): switch to smaller runners after done treoubleshooting
+        os: [ubuntu-latest-l, windows-latest-l, macos-13, macos-latest]
```

actionlint (1.7.4)

label "ubuntu-latest-l" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-22.04", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "macos-12-xl", "macos-12-xlarge", "macos-12-large", "macos-12", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)